<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreationTicket</class>
 <widget class="QDialog" name="CreationTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="comboBoxResultat">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>420</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogiciel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Logiciel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCategorie">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cat√©gorie</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLogiciel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelEasyTicket">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>72</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>EasyTicket</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>33</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNouveauTicket">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>103</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouveau ticket</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCategorie">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="champTitre">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelMessage">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>186</y>
     <width>103</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouveau ticket</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="champMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>876</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBoxResultat</sender>
   <signal>rejected()</signal>
   <receiver>CreationTicket</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxResultat</sender>
   <signal>accepted()</signal>
   <receiver>CreationTicket</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
